docs/setup.rst

```diff
@@ -79,6 +79,44 @@ Docker Compose
 Now proceed to `"Setup" <#setup>`__.
 
 
+Heroku
+------
```

Comment:
For other deployments we recommend using the upgrade script, as documented here: http://docs.redash.io/en/latest/upgrade.html.

The way it runs migrations is by comparing existing migrations with new ones. Which is awful and can't be done in stateless environments like Heroku.